inheritance/src/test/java/inheritance/LibraryTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package inheritance;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class LibraryTest {
+    @Test
+    public void testSomeLibraryMethod() {
+        Library classUnderTest = new Library();
+        assertTrue("someLibraryMethod should return 'true'", classUnderTest.someLibraryMethod());
+    }
+    Library.Restaurant testRestaurant;
+    @Before
+//    @Test
+    public void testRestaurant_constructor() {
+        testRestaurant = new Library.Restaurant("The Alchemist's Alembic", 5, 3);
+
+    }
+    @Test
+    public void testRestaurant_toString() {
+        assertEquals("String should match Restaurant test", "Name: The Alchemist's Alembic, 5 Stars and 3$ (of $$$$$)", testRestaurant.toString());
+    }
+
+    Library.Review testReview;
+    @Before
+    public void testReview_constructor() {
+        testReview = new Library.Review("This place is the best for drinking elven wine.", "Juniper Meadows", 5);
```

Comment:
This test is missing a `Restaurant name` argument and is preventing your code from compiling. Anytime you define a method it must received the parameters you said it would when you defined the method: `public Review(Restaurant name, String body, String author, int numberStars)`